doxygen/src/pack_publish.txt

```diff
@@ -1,75 +1,225 @@
+that helps you with both tasks: a
+<a class="el" href="https://github.com/Open-CMSIS-Pack/gen-pack" target="_blank">Bash library for gen-pack scripts</a>.
+Make sure that you have used it to validate your PDSC file and the pack contents before submitting it to any
+\ref cp_KeilComPack "pack indexing service". The library can be used on all major host operating systems and can also be run
+in GitHub actions.
+
+\section cp_LocalInstallation Local installation
 
-\section cp_LocalInstallation Local Installation
 The easiest way to distribute a software pack is attaching it to an email or using other means of electronic distribution
-(for example using a company internal file server). The installation of a software pack is usually triggered by
-double-clicking the Pack file. The pack management tool is invoked and the Pack is installed on the local machine. After
-installation, the Pack will be shown in the list of installed packs (tool dependent). If the PDSC file does not contain a
-valid \c \<url\> element, a manual installation is required to update a software pack. Otherwise, pack management tools can
-check for updates on the server or the local machine that is specified by \c \<url\>.
+(for example using a file server). Install the software pack on a PC using
+<a class="el" href="https://github.com/Open-CMSIS-Pack/cmsis-toolbox/blob/main/docs/build-tools.md#cpackget-details" target="_blank">cpackget</a>,
+the pack management tool that comes with the CMSIS-Toolbox:
+\code
+cpackget add vendor.packname.version.pack
+\endcode
+If the PDSC file does not contain a valid \c \elem{url} element, a
+manual installation is required to update a software pack. Otherwise, \c cpackget can check for updates on the server or the
+local machine that is specified by \c \elem{url}.
+
+\note For automatic updates on a local machine, specify the \c \elem{url} using the
+<a href="https://en.wikipedia.org/wiki/File_URI_scheme" target="_blank">file URI scheme</a>.
 
-\image html LocalInstallation.png "Local installation and update from URL"
+\section cp_WebDownload Web download
 
-\note For automatic updates on a local machine, specify the \c \<url\> using the
-<a href="https://en.wikipedia.org/wiki/File_URI_scheme" target="_blank">file URI scheme</a>.
+If you want to make your packs widely available, you need to host them on the web. For publishing packs, these
+options are available:
+- \ref cp_HowWebDownloadWorks
+- \ref cp_GitHubHosting
+
+The following diagram shows how you can publish your packs when using a \ref cp_KeilComPack "pack indexing service":
+
+\image html IndexingService.png "Pack hosting and indexing service"
+
+When you run the command \c cpackget \c update-index, the tool queries the \ref pidxFile "package index file (index.pidx)"
+from the pack index server. This file is generated using the \ref cp_vidx2pidx "vidx2pidx tool" from the
```

Comment:
Unfortunately we don't run vidx2pidx on the keil.com server.
Also I am not aware that vidx2pidx would run any schema check on pdsc files.
The vidx2pidx is really an offline utility to create dedicated index.pidx files based on a custom `*.vidx` file.